doc/graphs/paired_sample_t_statistic.svg

```diff
@@ -0,0 +1,46 @@
+<svg xmlns:xlink="http://www.w3.org/1999/xlink" width="13.643ex" height="6.676ex" style="vertical-align: -2.838;" viewBox="0 -1652.5 5874.2 2874.4" role="img" focusable="false" xmlns="http://www.w3.org/2000/svg" aria-labelledby="MathJax-SVG-1-Title">
```

Comment:
Looks like we still have the "ex" unit here.

Reply:
I thought the problem may have just been vertical align. All instance of ex have been removed and replaced with pixel counts in 0ac42ae.

Reply:
https://drafts.csswg.org/css-values-4/#font-relative-length   says

ex unit
Equal to the used x-height of the first available font [CSS3-FONTS]. The x-height is so called because it is often equal to the height of the lowercase "x". However, an ex is defined even for fonts that do not contain an "x". The x-height of a font can be found in different ways. Some fonts contain reliable metrics for the x-height. If reliable font metrics are not available, UAs may determine the x-height from the height of a lowercase glyph. One possible heuristic is to look at how far the glyph for the lowercase "o" extends below the baseline, and subtract that value from the top of its bounding box. In the cases where it is impossible or impractical to determine the x-height, a value of 0.5em ...

So it is like em units and should be a legal SVG unit.  I don't see why it should should cause trouble in generating the docs. Using mathJax has pulled in this new definition width="13.643ex".

But the xsltproc version we are using is old and probably predates this newer measure.  When we tried to use a newer version, the build process went wrong, so we have stuck to the old one.

But having said that, the default we have used elsewhere is pixels and that is the svg default, so it should work by just deleting the ex.  But it may cause some change in font size.  em might be closer?  Worth eyeballing the entire resulting docs - look at formulas?

Reply:
The font size changes if you just delete the ex off the end. I pulled up the SVG files with the original dimensions in a photo viewer to get the dimensions in pixels and then plugged them in. No issues with rendering that I see.